node/p2p/misc.go

```diff
@@ -0,0 +1,64 @@
+package p2p
+
+import (
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	connmgr "github.com/libp2p/go-libp2p-connmgr"
+	coreconnmgr "github.com/libp2p/go-libp2p-core/connmgr"
+	"github.com/libp2p/go-libp2p-core/peerstore"
+	"github.com/libp2p/go-libp2p-peerstore/pstoremem"
+	"github.com/libp2p/go-libp2p/p2p/net/conngater"
+)
+
+// ConnManagerConfig configures connection manager.
+type ConnManagerConfig struct {
+	// Low and High are watermarks governing the number of connections that'll be maintained.
+	Low, High int
+	// GracePeriod is the amount of time a newly opened connection is given before it becomes subject to pruning.
+	GracePeriod time.Duration
+}
+
+// DefaultConnManagerConfig returns defaults for ConnManagerConfig.
+func DefaultConnManagerConfig() *ConnManagerConfig {
+	return &ConnManagerConfig{
+		Low:         50,
+		High:        100,
```

Comment:
Doesn't this seem a bit too high? What are numbers for e.g. eth2, or for bitcoin?

Actually for full nodes this probably fine. Later we can have more lean `DefaultLightConnManagerConfig` or sth.

Reply:
For a full node that's totally fine